x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/CacheService.java

```diff
@@ -141,4 +183,123 @@ public void removeFromCache(final Predicate<CacheKey> predicate) {
         }
         cache.refresh();
     }
+
+    void setCacheSyncInterval(TimeValue interval) {
+        assert cacheSyncTask != null;
```

Comment:
I think this is a leftover from last iteration? Since it is a final field initialized in constructor, I would prefer not to have this assert.

Reply:
Right, I removed it